jaxrs/pom.xml

```diff
@@ -31,9 +31,9 @@
     <artifactId>jsonp-jaxrs</artifactId>
     <packaging>bundle</packaging>
     <version>1.2-SNAPSHOT</version>
-    <name>JSR 374 (JSON Processing) Media for JAX-RS</name>
-    <description>JAX-RS MessageBodyReader and MessageBodyWriter to support JsonValue API of JSR 374:Java API for Processing JSON</description>
-    <url>https://javaee.github.io/jsonp</url>
+    <name>Jakarta JSON Processing Media for JAX-RS</name>
+    <description>JAX-RS MessageBodyReader and MessageBodyWriter to support JsonValue API of Jakarta JSON Processing</description>
```

Comment:
shouldn't 'JAX-RS' be replaced with something like Jakarta RESTful Web Services here?

Reply:
Fixed